### DISCUSSION TITLE
Clash配置 | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#150] [Announcements]
https://blog.goalonez.site/blog/Clash%E9%85%8D%E7%BD%AE.html 

 Goalonez Blog

**Comment by JazZyzJ** [comment]
想问一下clash pro中的enhence 模式是不会管rule直接统统转proxy是吗()

**Comment by Goalonez** [comment]
> @JazZyzJ
> 想问一下clash pro中的enhence 模式是不会管rule直接统统转proxy是吗()

还是会走规则的
